modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/filters/implementation/SelectionFilter.tsx

```diff
@@ -193,7 +193,7 @@ function SelectionFilter({
 		// @ts-ignore
 
 		(value: string) => {
-			setSearchOptions({currentPage: 1, query: '', search: value});
+			setSearchOptions({currentPage: 1, query: value, search: value});
```

Comment:
This was preventing 'filtering' the options of the filter. It was only working for filters that retrieve the options from a URL (commerce).